cryptostore/data/arctic.py

```diff
@@ -52,6 +52,8 @@ def write(self, exchange, data_type, pair, timestamp):
         elif data_type == OPEN_INTEREST:
             df = df.drop(['symbol', 'feed'], axis=1)
             chunk_size = 'D'
+        elif data_type == LIQUIDATIONS:
```

Comment:
i think you can just change the elif above to this:

elif data_type == OPEN_INTEREST or data_type == LIQUIDATIONS

Reply:
Hi, @bmoscon I have updated the PR as you requested.